assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -197,7 +288,7 @@ const baseSelectors = {
 	 *   "dependencies": [
 	 *     "analytics"
 	 *   ],
-	 *   "dependants": []
```

Comment:
Whoops, that's probably my fault, I get my English-es confused sometimes 😅

Reply:
Nope, if this is wrong, it's on me :)

It is set to this on the server, so let's change it back for now since that's what will actually be returned. Let's rename it in a separate issue as there could be further implications on other code.